velox/expression/tests/utils/ArgumentTypeFuzzer.cpp

```diff
@@ -35,6 +38,92 @@ std::optional<TypeKind> baseNameToTypeKind(const std::string& typeName) {
   return tryMapNameToTypeKind(kindName);
 }
 
+namespace {
+
+bool isDecimalBaseName(const std::string& typeName) {
+  auto normalized = boost::algorithm::to_lower_copy(typeName);
+
+  return normalized == "decimal";
+}
+
+/// Returns true only if 'str' contains digits.
+bool isPositiveInteger(const std::string& str) {
```

Comment:
It seems there are several functions defined as isPositiveInteger and they are the same. Do we consider extracting it out for reuse? If yes, I'm glad to work on this change in a separate PR.